### PR TITLE
First draft of new onboarding guide for Programs (non-technical)

### PR DESCRIPTION
**Pull request by hicksjoseph**
First draft of new onboarding guide for Programs (non-technical).

### CODE REVIEW

File: docs/user-manual/onboarding-guide/programguide [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+- [ ] **Define user needs:** To create program forms that people will find intuitive and easy to use, consider the needs of both your residents and program administrators.  
+      - [ ] Spend time to understand how your residents interact with the program applications so that you can optimize the experience of CiviForm users (including residents and administrators):  
+            - [ ] Review [Preparing for Human-Centered Redesign](https://beeckcenter.georgetown.edu/report/preparing-for-human-centered-redesign/).  
+            - [ ] Read more about [mapping the applicant experience](https://github.com/usds/benefits-enrollment-prototype/blob/master/assets/discovery-findings-mapping-enrollment-Nov2016.pdf).  
+      - [ ] Consider engaging community-based organizations for user research.
+
+- [ ] **Decide who will be the [CiviForm Admin](https://docs.civiform.us/overview/how-does-civiform-work\#civiform-admin-experience)** across your civic entity  
+      - [ ] Decide who will own and maintain your CiviForm instance in the long term (IT team, Digital Services or other). This will be the person who creates the Forms and Programs on CiviForm.
```
Comment (dkatzz) [inline docs/user-manual/onboarding-guide/programguide:36]: **question:** Should this be a sub-bullet? Seems a bit unrelated to the parent.

File: docs/user-manual/onboarding-guide/programguide [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+- [ ] **Review and evaluate** the [Recommended CiviForm Launch Team](https://docs.google.com/document/d/1AacsuDbMmVfmBuWWSNRz3nOUYR3WG5\_KDnDgxrc6z0A/edit)  
+      - [ ] Consider if any new staffing or resources are needed for the implementation / maintenance of CiviForm and/or the programs.
+
+- [ ] **Choose programs to put in CiviForm:** Determine which programs will benefit most from using CiviForm (e.g., benefit applications, complaint forms, event registrations, permit applications).  
+      - [ ] Evaluate programs most likely to benefit from CiviForm by using these resources:  
+            - [ ]  [CiviForm Fellowship - Program Assessment](https://docs.google.com/document/d/10TTk68evp-X8iJ2W3S2--MpiqMeg1Uf0jTysbZITEdg/edit?usp=sharing)  
+            - [ ]  [CiviForm - Program Evaluation Worksheet ](https://docs.google.com/spreadsheets/d/1\_NNMln-LOAxxXCoYgh1oI0uaS6YhoyMoBvudaiKRMm4/edit?resourcekey=0-KPl3Y1YgUYty81OKHbRMpQ\#gid=629433048)
```
Comment (dkatzz) [inline docs/user-manual/onboarding-guide/programguide:44]: **nitpick:** Remove space after Worksheet

File: docs/user-manual/onboarding-guide/programguide [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+- [ ] **Choose programs to put in CiviForm:** Determine which programs will benefit most from using CiviForm (e.g., benefit applications, complaint forms, event registrations, permit applications).  
+      - [ ] Evaluate programs most likely to benefit from CiviForm by using these resources:  
+            - [ ]  [CiviForm Fellowship - Program Assessment](https://docs.google.com/document/d/10TTk68evp-X8iJ2W3S2--MpiqMeg1Uf0jTysbZITEdg/edit?usp=sharing)  
+            - [ ]  [CiviForm - Program Evaluation Worksheet ](https://docs.google.com/spreadsheets/d/1\_NNMln-LOAxxXCoYgh1oI0uaS6YhoyMoBvudaiKRMm4/edit?resourcekey=0-KPl3Y1YgUYty81OKHbRMpQ\#gid=629433048)  
+      - [ ] Examine selected programs to identify opportunities to streamline overlapping questions:  
+            - [ ] Review the information gathered on the application for each program and remove any unnecessary questions.  
+            - [ ] Work with program stakeholders including program administrators to understand the administrative and policy requirements for different programs.   
+            - [ ] Identify common questions across programs (Ex. Name, Address, income).
```
Comment (dkatzz) [inline docs/user-manual/onboarding-guide/programguide:48]: **nitpick:** Keep capitalization the same on Name, Address, income

File: docs/user-manual/onboarding-guide/programguide [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+      - [ ] For each program:  
+            - [ ] Decide on who the program leader/owner and primary point of contact will be.  
+            - [ ] Decide how applications to the program will be reviewed. Will they be reviewed in CiviForm, or pulled into an external system? If an external system, identify the technical PoCs.
+
+- [ ] **Join the CiviForm community**  
+      - [ ] Understand how the CiviForm community works.  
+            - [ ] [Community member responsibilities](https://docs.civiform.us/governance-and-management/governance/roles-committees-and-responsibilities)  
+            - [ ] [Governance Processes](https://docs.civiform.us/governance-and-management/governance/governance-processes)  
+      - [ ] Learn about  the [CiviForm community's communication channels](https://docs.civiform.us/governance-and-management/governance/communication).  
+            - [ ] Join [CiviForm Slack](https://civiform.slack.com/) and introduce yourself in the \#introductions channel.
```
Comment (dkatzz) [inline docs/user-manual/onboarding-guide/programguide:60]: **question:** Does the \ add formatting?

File: docs/user-manual/onboarding-guide/programguide [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+            - [ ] Monthly join the CiviForm Governance group to learn about best practices, new features and tools, as well as how other civic entities are leveraging CiviForm.
+
+## Preparation for Launch 
+
+- [ ] Review the [form building how-to-guide](https://docs.civiform.us/user-manual/civiform-admin-guide/working-with-programs/create-a-program) to begin building your first program  
+      - [ ] User-centric design: Create forms that are easy to understand and complete, using [plain language](http://plainlanguage.gov).  
+      - [ ] Keep  accessibility standards (e.g., WCAG) in mind when creating your forms.  
+      - [ ] Consider which languages Program questions and answers will be translated to and which languages [CiviForm supports natively](https://docs.civiform.us/user-manual/civiform-admin-guide/manage-translations-for-programs-and-questions).  
+            - [ ] Find translation resources and add time for translations to your project timeline.  
+      - [ ] Data validation: Implement program checks (such as phone number formats and hints to users “select only two choices” to prevent errors and ensure data accuracy.
```
Comment (dkatzz) [inline docs/user-manual/onboarding-guide/programguide:74]: **nitpick:** No close parentheses

File: docs/user-manual/onboarding-guide/programguide [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+- [ ] Decide on launch announcement plans to ensure that residents can find and apply for the new streamlined programs.  
+      - [ ] Identify target audience/user group, communication channels (eg. press release, website announcement, social media posts, a newsletter) to share the news with residents.  
+      - [ ] Develop a communication plan: Inform citizens and stakeholders about the new system and its benefits and consider how you may drive awareness about the program.  
+      - [ ] Consider relevant metrics to measure if you are reaching your residents.  
+- [ ] Generate excitement by recording demos and sharing any initial findings, positive data outcomes, or testimonials resulting from use of CiviForm to connect users to the program.   
+- [ ] **Announce CiviForm to your community**   
+- [ ] Put ample opportunities for feedback in place during and after the launch so that users can flag any questions or bugs.   
+- [ ] If programs will be rolled out in phases, provide users with a timeline of future planned launches for additional programs.
+
+## Next steps \- Ongoing Evaluation and Improvement
```
Comment (dkatzz) [inline docs/user-manual/onboarding-guide/programguide:97]: **question:** What does \- do for formatting?

File: docs/user-manual/onboarding-guide/programguide [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+
+- [ ] Performance metrics: Track key performance indicators (KPIs) to measure the impact of CiviForm.  
+- [ ] User feedback: Gather feedback from citizens and staff to identify areas for improvement.  
+- [ ] Iterative development: Continuously update and refine forms based on user needs and data analysis.  
+- [ ] Continue to grow CiviForm, adding new programs over time.
+
+## Check out these links to read more about CiviForm
+
+* CiviForm home page (To Be Linked)
```
Comment (dkatzz) [inline docs/user-manual/onboarding-guide/programguide:106]: **nitpick:** It may be helpful to add "TODO" with a bug number (if it exists) to help you find these.

**Review by dkatzz — APPROVED** [review]

File: docs/user-manual/onboarding-guide/programguide (hunk 1 above)
Comment (hicksjoseph) [reply docs/user-manual/onboarding-guide/programguide:36]: Thanks. I can't see which line you are referencing. Can you make a comment in the Google Doc for now?

File: docs/user-manual/onboarding-guide/programguide (hunk 2 above)
Comment (hicksjoseph) [reply docs/user-manual/onboarding-guide/programguide:44]: Thanks. Can you make a comment in the Google Doc for now?

File: docs/user-manual/onboarding-guide/programguide (hunk 3 above)
Comment (hicksjoseph) [reply docs/user-manual/onboarding-guide/programguide:48]: Thanks. I will change this in the Google Doc.

File: docs/user-manual/onboarding-guide/programguide (hunk 4 above)
Comment (hicksjoseph) [reply docs/user-manual/onboarding-guide/programguide:60]: This md was mostly outputted from Google Docs. I will have to check what the '\#' is supposed to do.

File: docs/user-manual/onboarding-guide/programguide (hunk 5 above)
Comment (hicksjoseph) [reply docs/user-manual/onboarding-guide/programguide:74]: Thanks. Can you make a comment in the Google Doc for now?

File: docs/user-manual/onboarding-guide/programguide (hunk 6 above)
Comment (hicksjoseph) [reply docs/user-manual/onboarding-guide/programguide:97]: The dash and [ ] is the markdown code for checkbox. Google Docs outputted this and it was the same as the checkboxes that you pointed me to. However, it doesn't display correctly on the website (maybe due to the layout and css), but does seem to display correctly on the docs site. I wanted to check this in to the docs site to see how it behaves.

File: docs/user-manual/onboarding-guide/programguide (hunk 3 above)
Comment (dkatzz) [reply docs/user-manual/onboarding-guide/programguide:48]: What is the plan to integrate the changes back into github? I imagine it will now be hard to keep track of what has changed in the google doc?

File: docs/user-manual/onboarding-guide/programguide (hunk 2 above)
Comment (dkatzz) [reply docs/user-manual/onboarding-guide/programguide:44]: This is specifically related to markdown, so it doesn't really apply in the google doc

Comment (hicksjoseph) [reply docs/user-manual/onboarding-guide/programguide:44]: Thanks. I will track down how the space got in there since it was automatically generated markdown.

File: docs/user-manual/onboarding-guide/programguide (hunk 3 above)
Comment (hicksjoseph) [reply docs/user-manual/onboarding-guide/programguide:48]: The short term plan (following your advice) is to have the content complete before uploading any changes to github. If an error makes it on the site, use github workflow to fix.

File: docs/user-manual/onboarding-guide/programguide (hunk 4 above)
Comment (hicksjoseph) [reply docs/user-manual/onboarding-guide/programguide:60]: The backslash escapes generates literal characters. In this case, it will generate a # instead of it being a comment.
